devices/RadioReceiver/Devices/Tea5767/FrequencyRange.cs

```diff
@@ -9,12 +9,12 @@ namespace Iot.Device.RadioReceiver
     public enum FrequencyRange
     {
         /// <summary>
-        /// 76MHz - 90MHz
+        /// 76MHz - 90MHz.
```

Comment:
Incosistent with other places - please check how you handled it here: devices/Bmxx80/StandbyTime.cs

Reply:
What do you mean?